devtools/credits.pl

```diff
@@ -1015,10 +1015,14 @@ sub add_paragraph {
 			end_section();
 
 			begin_section("Ultima");
-				add_person("Dominik Reichardt", "Dominus Dragon", "(Pentagram)");
-				add_person("Eric Fry", "yuv422", "(Nuvie)");
 				add_person("Paul Gilbert", "dreammaster", "");
+				add_person("Eric Fry", "yuv422", "(Nuvie)");
```

Comment:
I added users with 100+ commits to either project, based on Openhub (thanks Lightkey for pointing out I use that to check contribution totals). If any of the users have publically known names, let me know and I'll flesh out their entries